Objects/descrobject.c

```diff
@@ -1946,8 +1946,28 @@ ga_mro_entries(PyObject *self, PyObject *args)
     return PyTuple_Pack(1, alias->origin);
 }
 
+static PyObject *
+ga_instancecheck(PyObject *self, PyObject *Py_UNUSED(ignored))
+{
+    return PyErr_Format(PyExc_TypeError,
+                        "TypeError: Subscripted generics cannot be used with"
+                        " class and instance checks",
```

Comment:
I recommend not breaking the string literals -- I've come to the point where I prefer the occasional over-long line over not getting a grep hit on the text of an error message. And it's not *that* long...